.github/CONTRIBUTING.md

```diff
@@ -112,7 +112,7 @@ If you got an idea for a component that you think should be part of Kirby you sh
     - Create a pull request. If you created a draft PR during implementation you can just mark that as "ready for review".
     - Request that the pull request is code-reviewed.
     - Request that the pull request is [UX reviewed](#ux-review).
-    - When the pull request has been approved it will be automatically merged to `stable` via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
+    - When the pull request has been approved it will be automatically merged to `main` via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
```

Comment:
I don't think this is what we want to do: automerge PRs to `main`. The base branch should be `develop`. Does automerge work for _all_ branches?

Reply:
Will change to develop. Maybe we should get rid of the automerge part of this bullet, because people have to manually enable it on every PR right now?

Reply:
I was thinking of something similar 🙂